### DISCUSSION TITLE
There is a similar tool, broot, but about the pro/cons of both.

### DISCUSSION THREAD

**Discussion by hiimnooob** [sayanarijit/xplr#527] [General]
Take a quick look:

xplr
- written by Rust
- config with Lua
- default key binding is Vim
- nice [doc](https://xplr.dev/en/)

groot
- wriiten by Rust
- config with Hjson or TOML 
- default key binding isnt Vim (But i think it can)
- nice [doc](https://dystroy.org/broot/)

**Comment by sayanarijit** [comment]
Hi, thanks for posting. If you are looking for comparison, I'd say both are radically different ways of exploring files, catering the needs of different user requirements. I'd suggest try both in your day-to-day workflow. See what is missing, if something is missing, is it possible to implement yourself etc. If you are having trouble, try opening issues, or discussions like this for your specific requirements. You'll eventually see the pros/cons of each.

**Comment by erikLundstedt** [comment]
I personally use booth

for me, broot is used as an interactive replacement for the 'tree' command (I have in the past used a combo of some clever arguments to tree and fzf to achieve something similar, yet slower and harder to use) while xplr is used as a file manager/explorer for more day-to-day usage


while we are talking about alternatives, maybe have a look at w3m in file-tree mode 'w3m .'

**Reply by sayanarijit** [reply]
Nice. TIL w3m tree mode.

**Reply by hiimnooob** [reply]
sry i couldnt understand this line 

>  maybe have a look at w3m in file-tree mode 'w3m .'

**Comment by sayanarijit** [comment]
For the tree-view lovers: https://github.com/sayanarijit/tree-view.xplr
